components/expenses/ExpenseForm.js

```diff
@@ -213,6 +221,7 @@ const ExpenseFormBody = ({
   const [step, setStep] = React.useState(stepOneCompleted || isCreditCardCharge ? STEPS.EXPENSE : STEPS.PAYEE);
   // Only true when logged in and drafting the expense
   const [isOnBehalf, setOnBehalf] = React.useState(false);
+  const [showModal, setShowModal] = React.useState(false);
```

Comment:
There can be multiple modals in this component. Let's make this explicit:

```suggestion
  const [showResetModal, setShowResetModal] = React.useState(false);
```